### DISCUSSION TITLE
Custom Navbar Problems

### DISCUSSION THREAD

**Discussion by AngeloCG97** [facebook/docusaurus#4933] [General]
I am working on a project in which I customize the docusarusus navigation bar, for which I create my own navigation bar and load it from the Layoud, this works very well in all the custom pages that I have created as part of the site in the one I am working on, but in the documentation section that is generated by the default docusaurus themes it does not load the navigation bar that it creates if not the default navigation bar, but without elements only with the search bar since in the docusaurus .config.js I removed from themeConfig the default navigation bar configuration.

Does anyone know how I can make the document section load the nabvar that is applied on the rest of the site?

I have tried looking within docusarus components to determine where the navigation bar used in the docs section loads, but to no avail.

**Comment by slorber** [comment]
I'm unable to understand what you talk about. Please try to be clearer and show screenshots, code examples, public site URL, public github repo, repro using codesandox...
